tests/zap_helper_test.py

```diff
@@ -518,6 +518,28 @@ def test_load_session_error(self, session_mock, isfile_mock):
         with self.assertRaises(ZAPError):
             self.zap_helper.load_session(file_path)
 
+    @patch('zapv2.core.xmlreport')
+    @patch('zapcli.zap_helper.ZAPHelper.write_report')
```

Comment:
What do you think of making use of `mock_open` to mock the lower level file opening here rather than the write_report method?